content/en/workflows/access.md

```diff
@@ -0,0 +1,134 @@
+- provide an identity for workflow [audit trails][1]
+
+To create a service account for a workflow, you must have either the Datadog admin role, or a custom role with the **Service Account Write** permission. The service account you create adopts your role and permissions. For more information on service accounts and permissions, see [Service accounts][2].
+
+<div class="alert alert-info">The service account you create allows a workflow to be triggered automatically. However, individual actions in the workflow may require <a href="/workflows/connections/">Connections</a>.</div>
+
+### Associate a service account with a workflow
+
+You can dynamically create a service account for your workflow when you [add an automatic trigger][3].
+
+1. Click the **Create** button. The **Create Service Account** dialog opens.
+1. Use the dropdown to select a role for your service account user.
+1. Click **Create** to save the service account.
+
+When you run a workflow, the service account user resolves the connections defined in the workflow actions. Therefore, the service account user needs the `connections_resolve` permission. The Datadog Admin Role and the Datadog Standard Role include the `connections_resolve` permission.
+
+The **Create Service Account** dialog in Workflows populates the service account details as follows:
+- Name: `[WF]: <your-workflow-name>`
+- Email: the email address associated with your Datadog account
+
+### View service account details
+
+To view a service account's details:
+1. On your workflow canvas, click on the automated trigger.
+1. Next to the service account, click **View**.
+
+{{< img src="workflows/service-account-details1.png" alt="Service account details" style="width:60%;" >}}
+
+## Action credentials
+
+Because workflow [actions][4] connect with external software systems, you may need to authenticate your Datadog account to the corresponding integration. A workflow can run successfully only if every workflow action that requires authentication can verify the identity of your Datadog account.
+
+Workflow actions can be authenticated in two ways:
+- Credentials and permissions configured in the integration tile
+- Custom connection credentials
+
+### Integration tile credentials
+
+Credentials and account authentication that you set up in the following Datadog integration tiles automatically propagate to the corresponding actions in Workflows:
+- Jira
+- PagerDuty
+- Slack
+
+Configure the integration tiles by following instructions in [Datadog Integrations][5].
+
+### Connection credentials
+
+Use a custom [connection][6] to authenticate a [generic action][7] or any action for which the integration tile does not offer authentication.The workflow connection object allows you to provide credentials to connect to an integration. The parameters vary by integration.
+
+To connect to an arbitrary service, use the HTTP Connection type, and choose from two authentication options:
+- Token based authentication
```

Comment:
```suggestion
- Token-based authentication
```
We're a little inconsistent with this one in the docs, but I think "token-based" is a compound adjective here.